scripts/zones/Metalworks/npcs/Hungry_Wolf.lua

```diff
@@ -61,15 +81,19 @@ function onEventFinish(player,csid,option)
         player:addQuest(BASTOK,SMOKE_ON_THE_MOUNTAIN);
     elseif (csid == 0x01ad) then
         player:tradeComplete();
-        player:addGil(GIL_RATE*300)
-        player:messageSpecial(GIL_OBTAINED,GIL_RATE*300);
-      player:addTitle(HOT_DOG);
-        if (player:getQuestStatus(BASTOK,SMOKE_ON_THE_MOUNTAIN) == QUEST_ACCEPTED) then
-            player:addFame(BASTOK,30);
-            player:completeQuest(BASTOK,SMOKE_ON_THE_MOUNTAIN);
-        else
-            player:addFame(BASTOK,5);
-        end
+        player:addGil(GIL_RATE * 300);
+        player:messageSpecial(GIL_OBTAINED,GIL_RATE * 300);
+        player:addTitle(HOT_DOG);
+            if (player:getQuestStatus(BASTOK,SMOKE_ON_THE_MOUNTAIN) == QUEST_ACCEPTED) then
+                player:addFame(BASTOK,30);
+                player:completeQuest(BASTOK,SMOKE_ON_THE_MOUNTAIN);
+            else
+                player:addFame(BASTOK,5);
+            end
```

Comment:
indentation

Reply:
fixed

Reply:
also, this is part of an entirely different quest that i did not script nor check for consistency, i just left as it already was since i don't know the specifics of how it goes so don't test this as part of my PR. (fixed indents anyway) =)